eos-payg-ctl/eos-payg-ctl

```diff
@@ -33,6 +35,20 @@ ERROR_DOMAIN = "com.endlessm.Payg1.Error"
 DBUS_PROPERTIES_INTERFACE = "org.freedesktop.DBus.Properties"
 
 
+def __get_utc_from_boottime(timestamp):
+    #FIXME: Replace this with time.clock_gettime(time.CLOCK_BOOTTIME)
+    # once we have Python 3.7
+    # https://phabricator.endlessm.com/T25065
+    try:
+        current_boottime = int(subprocess.check_output('get-clock-boottime'))
```

Comment:
`time.clock_gettime(7)` would also have worked on Python 3.5

Reply:
IMO, that would be preferable enough to warrant dropping `get-clock-boottime` immediately.